src/metabase/channel/template/handlebars.clj

```diff
@@ -0,0 +1,55 @@
+(ns metabase.channel.template.handlebars
+  (:require
+   [clojure.walk :as walk]
+   [metabase.util :as u])
+  (:import
+   (com.github.jknack.handlebars
+    Handlebars Template)
+   (com.github.jknack.handlebars.cache
+    ConcurrentMapTemplateCache)
+   (com.github.jknack.handlebars.io
+    ClassPathTemplateLoader)))
+
+(set! *warn-on-reflection* true)
+
+(defn registry
+  "Create a new Handlebars instance with a template loader."
+  ^Handlebars [loader]
+  (doto (Handlebars. loader)
+    (.with (ConcurrentMapTemplateCache.))))
+
+(defn classpath-loader
+  "Create a ClassPathTemplateLoader with a prefix and postfix."
+  [prefix postfix]
+  (ClassPathTemplateLoader. prefix postfix))
+
+(def ^:private default-hbs
+  (registry (classpath-loader "/" "")))
```

Comment:
I decided to use an empty postfix as default here because I don't know what's the standard extension for handlebars, I see both .hbs and .handlebars